app/bundles/ChannelBundle/Command/SendChannelBroadcastCommand.php

```diff
@@ -50,6 +50,22 @@ protected function configure()
                         'id', 'i', InputOption::VALUE_OPTIONAL,
                         'The ID for a specifc channel to process broadcasts for pending contacts.'
                     ),
+                    new InputOption(
+                        'min-contact-id', 'minci', InputOption::VALUE_OPTIONAL,
```

Comment:
I don't think using multiple letters for the "short" syntax works because Symfony will kick an error saying that `The "-m" option does not exist.`